tiledb/sm/filesystem/azure.cc

```diff
@@ -99,9 +100,14 @@ Status Azure::init(const Config& config, ThreadPool* const thread_pool) {
       std::make_shared<azure::storage_lite::storage_account>(
           account_name, credential, use_https, blob_endpoint);
 
-  const int connection_count = 1;
+  // Construct the Azure SDK blob client with a concurrency level
+  // equal to 'thread_pool_->num_threads'. Internally, the client
+  // will allocate an equal number of libcurl sessions with
+  // 'curl_easy_init'. This ensures that our 'thread_pool_' threads
+  // will not block on the blob client's internal request queue
+  // unless the user is performing concurrent I/O on this instance.
   client_ = std::make_shared<azure::storage_lite::blob_client>(
```

Comment:
Noting here that we should eventually use the 2nd form of this function which takes a `ca_path` so we can control the cert location like we do for S3.

Reply:
Great catch -- I'll go ahead and add it now because we can easily fetch it with `global_state::GlobalState::GetGlobalState().cert_file();` as we already do in our own curl client.